src/sisl/io/vasp/chg.py

```diff
@@ -36,7 +36,7 @@ def read_grid(self, index=0, dtype=np.float64, **kwargs) -> Grid:
            For spin-polarized calculations, 0 and 1 refer to the charge (spin-up plus spin-down) and
            magnetitization (spin-up minus spin-down), respectively.
            For non-collinear calculations, 0 refers to the charge while 1, 2 and 3 to
-           the magnetization in the :math:`\sigma_1`, :math:`\sigma_2`, and :math:`\sigma_3` directions, respectively.
+           the magnetization in the :math:`\sigma_x`, :math:`\sigma_y`, and :math:`\sigma_z` directions, respectively.
```

Comment:
This is not generally correct, only when the user has `SAXIS = 0 0 1` in the VASP calculation (the default).

Reply:
ah! Thanks for always double checking my things! :) I will revert, and reference the `SAXIS` key.

Reply:
fixed!

Reply:
Perfect!